src/components/MessagesList/MessagesGroup/Message/MessagePart/AudioPlayer.vue

```diff
@@ -47,6 +48,13 @@ export default {
 			type: String,
 			default: '',
 		},
+		/**
+		 * File ID.
+		 */
+		id: {
```

Comment:
```suggestion
		messageId: {
```
I'd stay consistent with naming